src/render.c

```diff
@@ -70,7 +70,11 @@ void render_con(Con *con, bool render_fullscreen, bool already_inset) {
         }
         inset.height = 0;
         if (con->deco_rect.width != 0 && con->deco_rect.height != 0) {
-            con->deco_rect = rect_add(con->deco_rect, inset);
+            if (con_is_leaf(con)) {
+                con->deco_rect = rect_add(con->deco_rect, inset);
+            } else {
+                con->deco_rect = inset;
```

Comment:
I'm still not sure about this. In a `S[xterm T[xterm xterm]]` layout(*), why should `T`'s `deco_rect` be set like this?

*) `S` denotes a stacked container, `T` a tabbed one.

Reply:
I guess the else block is wrong. I wrote that just because it worked, not really for any good reason.